src/components/AllUsersTitle/style.js

```diff
@@ -0,0 +1,34 @@
+import { css } from "emotion"
+
+export const wrapper = css`
+  display: flex;
+  flex-flow: row wrap;
+  align-items: center;
+  justify-content: space-between;
+  list-style: none;
+  padding: 0;
+  margin-bottom: 48px;
+`
+
+export const h1Title = css`
+  font-weight: 400;
+  font-size: 3.36rem;
+  line-height: 1.12;
+  margin: 0;
+  padding: 0;
+`
+export const subTitle = css`
+  color: var(--global-grey-60);
+  font-size: 0.94rem;
+  line-height: 1.6;
+  margin: 0;
+  padding: 0;
+`
+export const btnLetters = css`
+  letter-spacing: 0.75px;
+  text-transform: uppercase;
+  font-weight: 700;
+  font-size: 1.35rem;
+  line-height: 1.39;
+  cursor: pointer;
```

Comment:
Why is cursor pointer needed here? Doesn't it automatically apply?
Also do we need this class to be applied on the span element or can we eliminate span completely and use this class on the Button component directly?